### PR TITLE
feat(curriculum): add registration form workshop to cert

### LINKED ISSUE #593: chore: add registration form workshop to main repo
We are using the existing registration form project found in the Responsive Web Design cert and adding that to the new frontend cert.
We are keeping the ids the same as the original ids from the RWD cert project.

**NOTE**
Please review the sheet for the lecture scripts. The lectures come before the workshops and some of the step descriptions will need to be updated. 

For example, if a lecture script already introduces a concept, then the language in the step shouldn't say "Now you are going to learn about X". It doesn't make sense to continuously repeat introductions, if it was already introduced just before this. 

You can update those steps to say "In the lecture videos, you learn how to work with X like this: [show example]" .
That makes sense.

At the end of the day, the flow of the curriculum needs to make sense to campers.  So we need to make sure the flow between the lecture videos, workshops and labs all make sense. 👍

### PR DESCRIPTION
**Pull request by jdwilkin4**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes https://github.com/freeCodeCamp/CurriculumExpansion/issues/593

### CODE REVIEW

**Review by zairahira — APPROVED** [review]

File: curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f02e7361b68405e27b62a5.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+demoType: onLoad
+---
+
+# --description--
+
+In this workshop, you will learn how to style forms by designing a registration form.
+
+All of the HTML boilerplate (`DOCTYPE`, `html`, `head`, and `body`) has been provided for you.
+
+Within the `body`, provide a heading context for the content, by adding an `h1` with the text `Registration Form`.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f02e7361b68405e27b62a5.md:15]:
```suggestion
Within the `body`, add a heading to give context to the form by using an `h1` element with the text `Registration Form`.
```

Comment (Sembauke) [reply curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f02e7361b68405e27b62a5.md:15]: It wasn't clear to me what I had to do here.

File: curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f1922fcbd2410527b3bd89.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+title: Step 3
+challengeType: 0
+dashedName: step-3
+---
+
+# --description--
+
+In the previous lecture videos, you learned how to work with viewport units.
+
+Remember that the `vh` unit stands for viewport height, and is equal to 1% of the `height` of the viewport. This makes it relative to the viewport height.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f1922fcbd2410527b3bd89.md:12]:
```suggestion
Remember, the `vh` unit stands for viewport height and represents 1% of the viewport's total height. This makes it relative to the viewport's height.
```

File: curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f5c3e399ff1a05629964e4.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+id: 60f5c3e399ff1a05629964e4
+title: Step 6
+challengeType: 0
+dashedName: step-6
+---
+
+# --description--
+
+As suggested by the title, you are creating a form. So, after the `p` element, insert a `form` with an `action` attribute targeting `https://register-demo.freecodecamp.org`.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f5c3e399ff1a05629964e4.md:10]:
```suggestion
As the title suggests, you are creating a form. After the `p` element, insert a `form` element with an `action` attribute that targets `https://register-demo.freecodecamp.org`.
```

File: curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f5d2776c854e069560fbe6.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+---
+id: 60f5d2776c854e069560fbe6
+title: Step 9
+challengeType: 0
+dashedName: step-9
+---
+
+# --description--
+
+The first `fieldset` will hold name, email, and password fields. Start by adding four `label` elements to the first `fieldset`.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f5d2776c854e069560fbe6.md:10]:
```suggestion
The first `fieldset` will contain the name, email, and password fields. Start by adding four `label` elements inside the first `fieldset`.
```

File: curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f85a62fb30c80bcea0cedb.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
+---
+id: 60f85a62fb30c80bcea0cedb
+title: Step 20
+challengeType: 0
+dashedName: step-20
+---
+
+# --description--
+
+Users will be allowed to choose either a `Personal` or `Business`.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f85a62fb30c80bcea0cedb.md:10]: This doesn't make much sense. I think there is some text missing.

```suggestion
Users will be able to choose whether the workshop is for `Personal` or `Business` purposes.
```

File: curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f8604682407e0d017bbf7f.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
+---
+id: 60f8604682407e0d017bbf7f
+title: Step 21
+challengeType: 0
+dashedName: step-21
+---
+
+# --description--
+
+Within each corresponding `label` element, and immediately after the `input` element, add a space and add the following text:
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-registration-form/60f8604682407e0d017bbf7f.md:10]:
```suggestion
Within each corresponding `label` element, and immediately after the `input` element, add a space followed by the following text:
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/25-front-end-development/workshop-registration-form/60ffe3936796ac04959285a9.md [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+title: Step 50
+challengeType: 0
+dashedName: step-50
+---
+
+# --description--
+
+Currently, if you type in the `input` or `textarea` elements, you will not be able to see the text. Also, their height is too small to be easy to use.
+
+Fix this, by setting the `color` to `#ffffff`, and setting their `min-height` to `2em`.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-registration-form/60ffe3936796ac04959285a9.md:12]:
```suggestion
Fix this by setting the `color` to `#ffffff`, and setting their `min-height` to `2em`.
```

File: curriculum/challenges/english/25-front-end-development/workshop-registration-form/6537e0be715fcb57d31ba8c3.md [hunk 8]
```diff
@@ -0,0 +1,290 @@
...
+id: 6537e0be715fcb57d31ba8c3
+title: Step 60
+challengeType: 0
+dashedName: step-60
+---
+
+# --description--
+
+Lastly change the text color of the `terms and conditions` link element to `#dfdfe2` by adding 
+a new selector in the CSS.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-registration-form/6537e0be715fcb57d31ba8c3.md:11]:
```suggestion
Lastly, change the text color of the `terms and conditions` link element to `#dfdfe2` by adding a new selector in the CSS.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**